### PR TITLE
Add in SwiftUI ViewModifier: .kerning()

### LINKED ISSUE #1567: [Term Entry] SwiftUI View Modifier: `.kerning()`
### Reason/inspiration (optional)

Please write a new entry under: content/swiftui/concepts/viewmodifier/terms/ about the modifier `.kerning()` in Swift.

A "Syntax" section with pseudo code snippet.
An "Example" section with a Swift code snippet and image to accompany. Images should go in the media/ folder.

To get started with the PR and entry file:

Under content/swiftui/concepts/viewmodifier/terms/, create a new folder called `kerning`. Then create a new file called `kerning.md` under content/swiftui/concepts/viewmodifier/terms/`kerning`

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by gracekishino**
### Description

- Added new folder `kerning` inside `content/swiftui/concepts/viewmodifier/terms/`.
- Added new term entry file `content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md` for SwiftUI ViewModifier: .kerning().
Added a new image for the SwiftUI ViewModifier: .kerning() term entry.

Closes #1567

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — COMMENTED** [review]
@gracekishino Looks good! I did a first review and had no comments. I'm passing this on to a second review.

**Review by SSwiniarski — APPROVED** [review]

File: content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                .kerning(0)
+            Text("STRETCHED TEXT")
+                .kerning(5)
+        }
+        .font(.largeTitle)
+    }
+}
+```
+
+In the above example, the `.kerning()` modifier is called on three text views, contained in a [`VStack`](https://www.codecademy.com/resources/docs/swiftui/views/vstack).
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md:48]:
```suggestion
In the above example, the `.kerning()` modifier is called on three text views, contained in a [`VStack`](https://www.codecademy.com/resources/docs/swiftui/views/vstack):
```

File: content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+Title: '.kerning()'
+Description: 'Sets the spacing size between text characters in a view.'
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md:3]:
```suggestion
Description: 'Adjusts the spacing size between text characters in a view.'
```

File: content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.kerning()`** modifier sets the spacing size between text characters in a [view](https://www.codecademy.com/resources/docs/swiftui/views).
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md:15]:
```suggestion
The **`.kerning()`** modifier adjusts the spacing size between text characters in a [view](https://www.codecademy.com/resources/docs/swiftui/views).
```

File: content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md (hunk 2 above)
Comment (yangc95) [reply content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md:3]: I recommend this verb

**Review by yangc95 — COMMENTED** [review]
@gracekishino Just a few quick suggestions!

**Comment by gracekishino** [comment]
@yangc95 thanks for the suggestions! Have updated the branch to reflect them.

File: content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.kerning()`** modifier adjusts the spacing size between text characters in a [view](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+```pseudo
+Text("This text will have kerning applied.")
+  .kerning(amount)
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/terms/kerning/kerning.md:21]:
```suggestion
    .kerning(amount)
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Nice! @gracekishino live link: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/kerning

Screen Shot 2023-03-05 at 8 51 33 AM

cc: @SSwiniarski

**Pull request merged**